Update changelog for v0.12.0 (#138)
mapbox/vision-ios-examples: CHANGELOG.md

## 1.0-32

> The latest version of the Vision Teaser App includes improvements in algorithms performance, camera calibration process and lane detection accuracy. Also the new version contains new improved ML-models for CV tasks like detection, segmentation and classification. The full list of changes:

Reviewer: ```suggestion
The latest version of the Vision Teaser includes improvements in algorithm performance, camera calibration process, and lane detection accuracy. The new version contains new improved ML-models for CV tasks like detection, segmentation, and classification. The full list of changes:
```

- `MapboxVision` v0.12.0

> - Added japanese traffic signs

Reviewer: ```suggestion
- Added detection of Japanese traffic signs
```

- Improved lanes detection algorithm
- Improved camera calibration algorithm

> - Utilized new ML models that reduce resource utilization

Reviewer: ```suggestion
- Utilized new ML models that reduce resource consumption
```
To remove "utilize" repetition.

## 1.0-32

The latest version of the Vision Teaser App includes improvements in algorithms performance, camera calibration process and lane detection accuracy. Also the new version contains new improved ML-models for CV tasks like detection, segmentation and classification. The full list of changes:
`MapboxVision` v0.12.0
- Added japanese traffic signs

> - Improved preformance

Reviewer 2: typo : `preformance`

Reviewer 2: Also IMO sounds vary vague

Author: changed to `Improved performance by utilizing ARM Advanced SIMD architecture extension`, is that better wording?

Reviewer 2: Well, I would expect not implementation side of the performance improvements but user facing. So if the SIMD optimizations reduced CPU usage probably it's good to mention `Reduced CPU usage of some feature`

Reviewer: Wait, this is release notes for teaser, who cares about SIMD and Neon? I think "Overall performance improvements" are good enough. Or if it seems vague, we can remove it completely.

Reviewer 3: I'd remove it completely (even `Performance improvements`). Definitely mentioning SIMD, Neon etc is not needed. I've just speeded up existing Neon a bit, it's not noticeable at all from user side.

Author: ok, removed

> - Improved lanes detection

Reviewer 4: nit:
```suggestion
- Improved lanes detection algorithm
```

- Improved camera calibration algorithm

> - Released new models version v4

Reviewer 2: Seems user does not know about versions of models - suggest to rephrase it. Maybe
`Utilized new ML models that reduce resource utilization`
